app/src/main/res/layout/topic_index_items.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical"
+    android:id="@+id/cardView"
+    android:layout_width="match_parent"
+    android:layout_height="230dp"
+    android:layout_margin="5dp"
+    app:cardCornerRadius="3dp">
+
+<LinearLayout
+    android:id="@+id/llContainer"
+    android:background="@color/light_brown"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <ImageView
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_margin="8dp"
+      android:src="@mipmap/ic_launcher"/>
+
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:background="@color/brown"
+      android:layout_marginTop="8dp"
+      android:padding="8dp"
+      android:orientation="vertical">
+    <TextView
```

Comment:
Keep it consistent. If you are keeping one extra blank line between views make sure you follow that here also.